nodewatcher/nodewatcher.py

```diff
@@ -244,11 +290,13 @@ def _poll_instance_status(config, scheduler_module, asg_name, hostname, instance
     :param instance_id: current instance id
     """
     _wait_for_stack_ready(config.stack_name, config.region, config.proxy_config)
+    _terminate_if_down(scheduler_module, config, asg_name, instance_id, minutes(3))
 
     idletime = _init_idletime()
     while True:
-        _store_idletime(idletime)
         time.sleep(60)
+        _store_idletime(idletime)
+        _terminate_if_down(scheduler_module, config, asg_name, instance_id, minutes(1))
```

Comment:
same here... maybe even make this user configurable

Reply:
I'll pull them out in constants and later we can make them configurable 👍